api-server/src/common/models/user.js

```diff
@@ -341,6 +341,21 @@ export default function initializeUser(User) {
     );
   };
 
+  User.prototype.mobileLoginByRequest = function mobileLoginByRequest(
+    req,
+    res
+  ) {
+    return new Promise((resolve, reject) =>
+      this.createAccessToken({ ttl: 77760000000 }, (err, accessToken) => {
```

Comment:
```suggestion
      this.createAccessToken(77760000000, (err, accessToken) => {
```
So, a couple of things.

We don't necessarily need to set this here.  We can modify `User.maxTTL` in https://github.com/freeCodeCamp/freeCodeCamp/blob/1155e8f2de435a3f43c6086e62a855226e15bea4/api-server/src/common/models/user.js#L164 

These should be _seconds_ not milliseconds.

cc @raisedadead

Reply:
I am unsure if the `User.settings.ttl` even works. I would not rely on it.

Reply:
I would not rely on it, pretty sure it's not used.